packages/react-chat/src/contexts/RuntimeContext/useRuntimeState.ts

```diff
@@ -35,15 +37,17 @@ export const useRuntimeState = ({ assistant, config }: Settings) => {
 
   const [indicator, setIndicator] = useState(false);
 
-  // TODO: refactor no-reply system
-  const [lastInteractionAt, setLastInteractionAt] = useState<number | null>(Date.now());
-  const [noReplyTimeout, setNoReplyTimeout] = useState<number | null>(null);
+  const { clearNoReplyTimeout, setNoReplyTimeout } = useNoReply(() => ({ interact, isStatus }));
```

Comment:
I can't directly reference, and have to use this function call `() => ({ interact, isStatus })` because `interact` and `isStatus` aren't declared yet.